lib/processors/bundlers/moduleBundler.js

```diff
@@ -15,12 +15,12 @@ const log = require("@ui5/logger").getLogger("builder:processors:bundlers:module
  * which the bundle module is loaded.
  * 	</li>
  *	<li>
- *		<code>raw</code>: A raw section determines the set of modules that should be embedded, sorts them according
+ *		<code>raw</code>: A 'raw' section determines the set of modules that should be embedded, sorts them according
  *		to their dependencies and writes them out 1:1 without any transformation or wrapping (raw). Only JavaScript
  *		sources can be embedded in a raw section.
  *	</li>
  *	<li>
- *		<code>preload</code>: A preload section packages resources that should be stored in the preload cache in the
+ *		<code>preload</code>: A 'preload' section packages resources that should be stored in the preload cache in the
  *		client. They can embed any textual resource type (JavaScript, XML, JSON and .properties files) that the
  *		bundling supports. UI5 modules are wrapped into a 'sap.ui.predefine' call. Other JavaScript modules will be
  *		embedded into a 'jQuery.sap.registerPreload' call, unless the asynchronous ui5loader is used. With the
```

Comment:
"Other JavaScript modules will be embedded into a 'jQuery.sap.registerPreload' call, unless the asynchronous ui5loader is used."

This should be 
"Other JavaScript modules will be embedded into a 'jQuery.sap.registerPreload' call, or in a "sap.ui.require.preload" call when the ui5loader is available."

In general, we might want to swap the descriptions and first describe the modern case (ui5loader available) and only afterwards the use case w/o ui5loader.

Reply:
Done